<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Whiteboard</class>
 <widget class="QWidget" name="Whiteboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_pen_color">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
background-color: rgb(50, 54, 63);
color: white; 
font:bold;
border: 0px;
padding-top: 1px;
padding-left: 4px;
padding-right: 0px;
}

QComboBox QListView {
	color: white;
}

QComboBox:hover {
	background-color: rgba(0, 127, 177, 0.6);
}

QComboBox::drop-down {
   border: 0px;
}

QComboBox::down-arrow {
	image: url(:/resources/img/down-arrow.png);
}
</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_pen_size">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
background-color: rgb(50, 54, 63);
color: white; 
font:bold;
border: 0px;
padding-top: 1px;
padding-left: 4px;
padding-right: 0px;
}

QComboBox QListView {
	color: white;
}

QComboBox:hover {
	background-color: rgba(0, 127, 177, 0.6);
}

QComboBox::drop-down {
   border: 0px;
}

QComboBox::down-arrow {
	image: url(:/resources/img/down-arrow.png);
}
</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>4</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>2px</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4px</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6px</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8px</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10px</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="save_whiteboard_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
background-color: rgb(50, 54, 63);
color: white; 
font:bold;
border: 0px;
}

QToolButton:hover {
	background-color: rgba(0, 127, 177, 0.6);
}</string>
       </property>
       <property name="text">
        <string>Save Whiteboard</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>247</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
